### ISSUE TITLE
Question: DVR and HLS on Safari

### ISSUE THREAD

**Issue by busla** [clappr/clappr#1358]
**Browser**: Safari Version 9.1.2 (11601.7.7)

**OS**: Mac OS 10.11.6 El Capitan

**Clappr Version**: FlasHLS Clappr (version: 0.4.15 - flashls: 0.4.4.22, id: o14)

**Steps to reproduce**:
I´ve read all the related issues regarding Safari and DVR support but can´t figure out if Clappr supports it or how users are resolving that issue. It would be great to have this added to the README in layman terms instead of a link to an issue.

I see that **hls.js** is integrated into Clappr by default and supports DVR in Safari on there demo page: http://video-dev.github.io/hls.js/demo/

For some reason I only get a stop button and no DVR.

<img width="708" alt="screen shot 2017-04-08 at 13 07 59" src="https://cloud.githubusercontent.com/assets/3162968/24829211/64242ed2-1c5d-11e7-9ed9-1c2981521392.png">

**Instantiation:**
```javascript
var player = new Clappr.Player({
  source: "http://ruvruv-live.hls.adaptive.level3.net/ruv/ruv/index.m3u8", 
  parentId: "#player",
  actualLiveTime: true,
  autoPlay: true,
  plugins: [LevelSelector, Clappr.FlasHLS],
  language: 'is-IS',
  strings: {
    'is-IS': {
      'live': 'Í beinni',
      'back_to_live': 'Aftur í beina útsendingu',
      'playback_not_supported': 'Stuðning vantar!'
     }
  }  
});
```

**Scripts loaded in my HTML:**

```html
  <script type="text/javascript" src="https://cdn.jsdelivr.net/clappr/latest/clappr.min.js"></script>
  <script type="text/javascript" src="http://cdn.jsdelivr.net/clappr.level-selector/latest/level-selector.min.js"></script>
  <script src="custom.js"></script>
```

**Did you try to reproduce this issue at http://cdn.clappr.io/?**
Yes, and there doesn´t seem to be DVR support in Safari, atleast not with the stream I´m using.

I understand that this is a duplicate but the resolutions are unclear to me. Please apologize 😄 

All the best!

**Comment by kslimani** [comment]
Hi @busla 
The provided source `http://ruvruv-live.hls.adaptive.level3.net/ruv/ruv/index.m3u8` returns a 401 Unauthorized HTTP response.

On MAC OS X with Safari browser, if you load an HLS video source, Clappr default behaviour is to use native HTML5 video. (_on other browsers like Chrome or Firefox, it use hls.js javascript library_).

On provided example, you configured Clappr to use internal Flash HLS plugin for HLS source playback.
This plugin should properly handle DVR (_if the HLS source has DVR capabilities_).

Can you try the following code on [Clappr demo page](http://clappr.io/demo/) ? :

```javascript
var playerElement = document.getElementById("player-wrapper");

var player = new Clappr.Player({
  source: 'https://wowza.jwplayer.com/live/jelly.stream/playlist.m3u8?DVR',
  plugins: [Clappr.FlasHLS],
  height: 360,
  width: 640
});

player.attachTo(playerElement);
```

**Comment by busla** [comment]
Thanks for the quick reply 😄 

Weird that you get a 401 on that URL. It might be blocked outside of Iceland.

Here is the global stream. Could you try that one?

http://ruvruverl-live.hls.adaptive.level3.net/ruv/ruverl/index.m3u8


Tried your code but still no DVR in Safari.

<img width="675" alt="screen shot 2017-04-08 at 13 42 48" src="https://cloud.githubusercontent.com/assets/3162968/24829448/5a037dd2-1c61-11e7-94d3-8982dc046082.png">


Also tried the demo page, but no DVR support:

![screen shot 2017-04-08 at 13 44 46](https://cloud.githubusercontent.com/assets/3162968/24829463/9a37548c-1c61-11e7-9ef4-0ccc8347dc7a.png)

**Comment by kslimani** [comment]
I tried demo page with `http://ruvruverl-live.hls.adaptive.level3.net/ruv/ruverl/index.m3u8` HLS source and DVR work (_with and without Flash HLS_), but i am on Windows 7. (_i will be able to test on MAC OS X on monday_). But it should work.

This question may be "dumb" : but do you have Flash plugin installed ? can you try again, and inspect the DOM to ensure there is a Flash embed loaded ?

If Flash HLS plugin is properly loaded, you should also see lines in javascript console like :

```
INFO:FlasHLS Clappr (version: 0.4.15 - flashls: 0.4.4.22, id: o53)
[...]
```

**Comment by busla** [comment]
Hey,

The DVR works fine on Chrome and Firefox, but not on Safari, as described in the original post.

The FlasHLS loads properly, I actually used that in the original post to get the version numbers 😄 

Thanks for looking into this.

**Comment by kslimani** [comment]
Without Flash HLS plugin, on MAC OSX with Safari, it is native HTML5 video which is used for playback. (_i don't remember how DVR playback is handled by Safari_).

**Comment by towerz** [comment]
The HTML5 video playback currently does not support DVR, and since we didn't find any other issue with using hls.js on Safari, it's now enabled by default.
